BILLthebuilder/automart
# #166628330 Implement LFA feedback on UI

#### What does thi PR do?
* It implements feedback I recieved from my LFA 
#### Description of Task to be completed?
* Add images to my UI template pages
* Recreate the post-ad.html page
#### How should this be manually tested?
* Clone the repository
* Checkout to the `ch-lfa-feedback-UI-166628330`branch
* Navigate to the `UI` or `UI/views` directories
* Open `index.html` or `views/post-ad.html` in your favorite browser and you will see images and minor changes that have been added
#### What are the relevant pivotal tracker stories?
* [#166628330](https://www.pivotaltracker.com/story/show/166628330 )
#### Screenshots
![index](https://user-images.githubusercontent.com/24655101/59355970-a81e1080-8d30-11e9-9a56-0cde64186328.png)
![post-ad](https://user-images.githubusercontent.com/24655101/59356015-ba984a00-8d30-11e9-93fa-352fd7929ecc.png)
![view-posted](https://user-images.githubusercontent.com/24655101/59356077-d00d7400-8d30-11e9-85fc-c971df7ac550.png)
